tests/test_fixtures/great_expectations_v013_titanic.yml

```diff
@@ -43,28 +43,6 @@ stores:
       class_name: TupleFilesystemStoreBackend
       base_directory: checkpoints/
 
-validation_operators:
```

Comment:
The only validation operator that existed was the `ActionListValidationOperator` - now that we've removed the legacy checkpoint, we can remove this entire module. This includes removing any references within YAML